client/web/src/cody/chat/CodyChatPage.tsx

```diff
@@ -39,25 +32,31 @@ import { ChatUI } from '../components/ChatUI'
 import { HistoryList } from '../components/HistoryList'
 import { useChatStore } from '../stores/chat'
 
-import { CodyPageIcon } from './CodyPageIcon'
+import { CodyColorIcon } from './CodyPageIcon'
 
 import styles from './CodyChatPage.module.scss'
 
 interface CodyChatPageProps {
     authenticatedUser: AuthenticatedUser | null
 }
 
+const VsCodeExtensionThumbnail = lazyComponent(() => import('./VsCodeExtensionThumbnail'), 'VsCodeExtensionThumbnail')
+const AskCodyThumbnail = lazyComponent(() => import('./AskCodyThumbnail'), 'AskCodyThumbnail')
+
 export const CodyChatPage: React.FunctionComponent<CodyChatPageProps> = ({ authenticatedUser }) => {
     const { reset, clearHistory } = useChatStore({ codebase: '' })
     const [enabled] = useFeatureFlag('cody-web-chat')
-    const onInstallClick = (): void => eventLogger.log(EventName.TRY_CODY_VSCODE)
-    const onMarketplaceClick = (): void => eventLogger.log(EventName.TRY_CODY_MARKETPLACE)
+    const [showVSCodeCTA, ____] = useState<boolean>(Math.random() < 0.5)
```

Comment:
You can just ignore the second value I think!

```suggestion
    const [showVSCodeCTA] = useState<boolean>(Math.random() < 0.5)
```

Reply:
done